src/components/PostList/PostList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo/PostInfo';
 import './PostList.scss';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  preparedPosts: Post[],
+};
+
+export const PostList: React.FC<Props> = ({ preparedPosts }) => (
+  <div className="posts">
+    {preparedPosts.map(({
+      title, body, user, comments,
+    }) => (
+      <PostInfo title={title} body={body} user={user} comments={comments} />
```

Comment:
this will work the same

```suggestion
    {posts.map((post) => (
      <PostInfo {...post} />
```